etl/steps/data/garden/wash/2024-01-06/who.meta.yml

```diff
@@ -0,0 +1,498 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Clean Water & Sanitation
+        - Clean Water
+        - Sanitation
+    display:
+      numDecimalPlaces: 1
+    processing_level: major
+  residence: |-
+    <% if residence == "Rural" %>rural <% elif residence == "Urban" %>urban <% elif residence == "Total" %><%- endif -%>
+  residence_pop: <% if residence == "Rural" %>rural <% elif residence == "Urban" %>urban <% elif residence == "Total" %>total <%- endif -%>
+  hygiene_basic_description_short: &hygiene_basic_desc_short |
+    Share of the {definitions.residence} population with a handwashing facility with soap and water at home.
+  hygiene_limited_description_short: &hygiene_limited_desc_short |
+    Share of the {definitions.residence} population with a handwashing facility lacking soap and/or water at home.
+  hygiene_no_facility_description_short: &hygiene_no_facility_desc_short |
+    Share of the {definitions.residence} population with no handwashing facility at home.
+  sanitation_basal_description_short: &sanitation_basal_desc_short |
+    Share of the {definitions.residence} population using at least improved sanitation facilities that are not shared with other households. Improved sanitation facilities are those designed to hygienically separate excreta from human contact, and include: flush/pour flush toilets connected to piped sewer systems, septic tanks or pit latrines; pit latrines with slabs (including ventilated pit latrines), and composting toilets.
```

Comment:
this is up to you but I think the titles of these indicators will likely be self-explanatory and you don't need to repeat the "`Share of the {definitions.residence} population using at least improved sanitation facilities that are not shared with other households.`". Would make the subtitles a bit shorter :)